wca-regulations.md

```diff
@@ -131,7 +131,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3h2) Puzzles whose colored plastic is visible inside the puzzle (e.g. "stickerless" puzzles) are permitted. This does not include the following puzzles:
         - 3h2a) Puzzles with transparent parts that can reveal more information about the state of the puzzle. Exception: An overlay sticker or a logo (see [Regulation 3l](regulations:regulation:3l)).
     - 3h3) Any modifications to a puzzle that result in poor performance by a competitor are not grounds for additional attempts.
-    - 3h4) For Clock, custom "inserts" (the same shape and size as the traditional paper inserts) are permitted, at the discretion of the WCA Delegate. The inserts must have a clear indication of 12 o'clock that matches the original inserts.
+    - 3h4) For Clock, the following modifications are permitted:
+        - 3h4a) Customs "inserts" (the same shape and size as the traditional paper inserts) are permitted, at the discretion of the WCA Delegate. The inserts must have a clear indication of 12 o'clock that matches the original inserts.
```

Comment:
The wording of this regulation assumes what traditional paper inserts are. Specifically, It would seem something like the Angstrom Clock would not be allowed due to the different shape of the insert. https://www.thecubicle.com/products/angstrom-temporus-clock

Reply:
Hey Jacob, 3h4a is basically the current 3h4 reorganized, so I don't see any problem on allowing something that is currently allowed